### PR TITLE
i18n(es): Updated docsearch & nav & ui

### PR DESCRIPTION
**Pull request by Zastinian**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- The i18n of the docsearch, nav and ui sections have been updated for the Spanish language.

#### Commits

[`7b8f036`](https://github.com/Zastinian/astro-docs/commit/7b8f036941c6e3b7f5f6ef18d966873422701b75)
[`4964193`](https://github.com/Zastinian/astro-docs/commit/49641936b9aa5d7ec25422711895f79f78495e92)

### CODE REVIEW

**Comment by Waxer59** [comment]
Hi thanks for the PR 🚀!. I recommend that as this PR is an update you reference the comits by putting the links. I give you an example 
```md
[`<id of the commit>`](<url>)
```
. This way we can better review the changes made

File: src/i18n/es/ui.ts [hunk 1]
```diff
@@ -1,113 +1,122 @@
...
-	'404.linkText': 'Volver a la página principal.',
-	// `<CodeSnippet>` vocabulary
-	'codeSnippet.terminalCaption': 'Ventana del terminal',
-	// `<LanguageSelect>` vocabulary
+	'404.linkText': 'Llévame a casa.',
+	// Etiquetas predeterminadas del componente Aside
+	'aside.note': 'Nota',
+	'aside.tip': 'Consejo',
+	'aside.caution': 'Precaución',
+	'aside.danger': 'Peligró',
```
Comment (Waxer59) [inline src/i18n/es/ui.ts:76]:
```suggestion
	'aside.danger': 'Peligro',
```

File: src/i18n/es/ui.ts [hunk 2]
```diff
@@ -1,113 +1,122 @@
...
+	// Vocabulario de <CodeSnippet>
+	'codeSnippet.terminalCaption': 'Ventana de terminal',
+	// Vocabulario de <LanguageSelect>
 	'languageSelect.label': 'Seleccionar idioma',
-	// Integrations vocabulary
+	// Vocabulario de integraciones
 	'integrations.changelog': 'Registro de cambios',
 	'integrations.footerTitle': 'Más integraciones',
-	'integrations.renderers': 'UI Frameworks',
+	'integrations.renderers': 'Frameworks de IU',
```
Comment (Waxer59) [inline src/i18n/es/ui.ts:84]:
```suggestion
	'integrations.renderers': 'Frameworks UI',
```

File: src/i18n/es/docsearch.ts [hunk 3]
```diff
@@ -4,5 +4,9 @@ export default DocSearchDictionary({
 	button: 'Buscar',
 	placeholder: 'Buscar en la documentación',
 	shortcutLabel: 'Presiona / para buscar',
+	resultsFooterLede: '¿Buscas una integración o tema de Astro? ¿Necesitas más ayuda?',
+	resultsFooterIntegrations: 'Directorio de integraciones de Astro',
+	resultsFooterThemes: 'Showcase de temas de Astro',
```
Comment (Waxer59) [inline src/i18n/es/docsearch.ts:9]:
```suggestion
	resultsFooterThemes: 'Galeria de temas de Astro',
```

File: src/i18n/es/nav.ts [hunk 4]
```diff
@@ -1,69 +1,123 @@
...
-	'guides/testing': 'Pruebas',
-	'guides/troubleshooting': 'Solución de problemas',
+	{ text: 'Guías', header: true, type: 'learn', key: 'features' },
+	{
+		text: 'Sintaxis de Plantilla de Astro',
+		slug: 'core-concepts/astro-syntax',
+		key: 'core-concepts/astro-syntax',
+	},
+	{
+		text: 'Frameworks de Interfaz de Usuario',
```
Comment (Waxer59) [inline src/i18n/es/nav.ts:47]:
```suggestion
		text: 'Frameworks UI',
```

File: src/i18n/es/nav.ts [hunk 5]
```diff
@@ -1,69 +1,123 @@
...
+	{ text: 'Assets (Experimental)', slug: 'guides/assets', key: 'guides/assets' },
+	{ text: 'Imágenes', slug: 'guides/images', key: 'guides/images' },
+	{ text: 'Fuentes', slug: 'guides/fonts', key: 'guides/fonts' },
+	{ text: 'Importaciones', slug: 'guides/imports', key: 'guides/imports' },
+	{
+		text: 'Renderizado en el lado del servidor (SSR)',
+		slug: 'guides/server-side-rendering',
+		key: 'guides/server-side-rendering',
+	},
+	{ text: 'Puntos Finales', slug: 'core-concepts/endpoints', key: 'core-concepts/endpoints' },
```
Comment (Waxer59) [inline src/i18n/es/nav.ts:73]:
```suggestion
	{ text: 'Endpoints', slug: 'core-concepts/endpoints', key: 'core-concepts/endpoints' },
```

File: src/i18n/es/ui.ts [hunk 6]
```diff
@@ -1,113 +1,122 @@
...
+		'Esta página aún no está disponible en tu idioma, así que te mostramos la versión en inglés. ¡Puedes ayudar traduciéndola!',
+	'fallbackContent.linkText': 'Obtén más información sobre cómo puedes contribuir',
+	// Página 404
 	'404.title': 'No encontrado',
 	'404.content': 'Esta página no está en nuestro sistema solar.',
-	'404.linkText': 'Volver a la página principal.',
-	// `<CodeSnippet>` vocabulary
-	'codeSnippet.terminalCaption': 'Ventana del terminal',
-	// `<LanguageSelect>` vocabulary
+	'404.linkText': 'Llévame a casa.',
```
Comment (Waxer59) [inline src/i18n/es/ui.ts:71]: yo lo dejaroa como estaba originalmente
```suggestion
	'404.linkText': Volver a la página principal.',
```

File: src/i18n/es/ui.ts [hunk 7]
```diff
@@ -1,113 +1,122 @@
...
+	'leftSidebar.learnTab': 'Aprender',
+	'leftSidebar.referenceTab': 'Referencia',
 	'leftSidebar.viewInEnglish': 'Ver en inglés',
 	'leftSidebar.sponsoredBy': 'Patrocinado por',
-	// Right Sidebar
+	// Barra lateral derecha
 	'rightSidebar.a11yTitle': 'Secundario',
 	'rightSidebar.onThisPage': 'En esta página',
-	'rightSidebar.overview': 'Sinopsis',
+	'rightSidebar.overview': 'Visión general',
```
Comment (Waxer59) [inline src/i18n/es/ui.ts:22]: Creo que este cambio no es necesario, personalmente lo dejaria como estaba anteriormente
```suggestion
	'rightSidebar.overview': 'Sinopsis',
```

File: src/i18n/es/nav.ts [hunk 8]
```diff
@@ -1,69 +1,123 @@
...
-	examples: 'Recetas',
-	'guides/migrate-to-astro': 'Migrar a Astro',
-	'guides/cms': 'Conectando un CMS',
-	'guides/integrations-guide': 'Agrega integraciones',
-	'guides/deploy': 'Desplegar',
-	'guides/recipes': 'Más recetas',
+	{ text: 'Recetas', header: true, type: 'learn', key: 'examples' },
+	{ text: 'Migrar a Astro', slug: 'guides/migrate-to-astro', key: 'guides/migrate-to-astro' },
+	{ text: 'Conectar un CMS', slug: 'guides/cms', key: 'guides/cms' },
+	{ text: 'Agregar Servicios Backend', slug: 'guides/backend', key: 'guides/backend' },
```
Comment (Waxer59) [inline src/i18n/es/nav.ts:35]:
```suggestion
	{ text: 'Agregar servicios backend', slug: 'guides/backend', key: 'guides/backend' },
```

File: src/i18n/es/nav.ts [hunk 9]
```diff
@@ -1,69 +1,123 @@
...
+	{ text: 'Agregar Integraciones', slug: 'guides/integrations-guide', key: 'guides/integrations-guide' },
```
Comment (Waxer59) [inline src/i18n/es/nav.ts:36]:
```suggestion
	{ text: 'Agregar integraciones', slug: 'guides/integrations-guide', key: 'guides/integrations-guide' },
```

File: src/i18n/es/nav.ts [hunk 10]
```diff
@@ -1,69 +1,123 @@
...
+	{ text: 'Desplegar tu Sitio', slug: 'guides/deploy', key: 'guides/deploy' },
```
Comment (Waxer59) [inline src/i18n/es/nav.ts:37]:
```suggestion
	{ text: 'Desplegar tu sitio', slug: 'guides/deploy', key: 'guides/deploy' },
```

File: src/i18n/es/nav.ts [hunk 11]
```diff
@@ -1,69 +1,123 @@
...
+	{ text: 'Más Recetas', slug: 'recipes', key: 'guides/recipes' },
```
Comment (Waxer59) [inline src/i18n/es/nav.ts:38]:
```suggestion
	{ text: 'Más recetas', slug: 'recipes', key: 'guides/recipes' },
```

File: src/i18n/es/nav.ts [hunk 12]
```diff
@@ -1,69 +1,123 @@
...
+	{ text: 'Conceptos básicos', header: true, type: 'learn', key: 'basics' },
 
-	//basics
-	basics: 'Fundamentos',
-	'core-concepts/project-structure': 'Estructura de proyecto',
-	'core-concepts/astro-components': 'Componentes',
-	'core-concepts/astro-pages': 'Páginas',
-	'core-concepts/layouts': 'Plantillas de página',
+	{
+		text: 'Estructura del Proyecto',
```
Comment (Waxer59) [inline src/i18n/es/nav.ts:20]:
```suggestion
		text: 'Estructura del proyecto',
```

File: src/i18n/es/ui.ts [hunk 13]
```diff
@@ -1,113 +1,122 @@
...
+	'recipesLink.plural': 'Recetas relacionadas',
+	// Texto de respaldo para <ContributorList>
+	'contributors.seeAll': 'Ver todos los colaboradores',
+	// Aviso de contenido de respaldo que se muestra cuando una página aún no está traducida
 	'fallbackContent.notice':
-		'Esta página aún no está disponible en tu idioma, mientras tanto verás la versión en inglés. ¡Ayúdanos a traducirla!',
-	'fallbackContent.linkText': 'Más información sobre cómo contribuir',
-	// 404 Page
+		'Esta página aún no está disponible en tu idioma, así que te mostramos la versión en inglés. ¡Puedes ayudar traduciéndola!',
+	'fallbackContent.linkText': 'Obtén más información sobre cómo puedes contribuir',
```
Comment (Waxer59) [inline src/i18n/es/ui.ts:67]:
```suggestion
	'fallbackContent.linkText': 'Obtén más información sobre cómo contribuir',
```

**Review by Waxer59 — COMMENTED** [review]
Te dejo unas sugerencias 🤠

**Comment by Zastinian** [comment]
> Te dejo unas sugerencias 🤠

Listo, estuve viendo y lo actualicé

Aquí te dejo los 2 commits que realice

[`7b8f036`](https://github.com/Zastinian/astro-docs/commit/7b8f036941c6e3b7f5f6ef18d966873422701b75)
[`4964193`](https://github.com/Zastinian/astro-docs/commit/49641936b9aa5d7ec25422711895f79f78495e92)

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Pull request merged**